tests/unit/amo/api/test_users.js

```diff
@@ -252,4 +253,37 @@ describe(__filename, () => {
       mockApi.verify();
     });
   });
+
+  describe('unsubscribeNotification', () => {
+    it('unsubscribes a user from a notification', async () => {
+      const { state } = dispatchClientMetadata();
+      const hash = 'some-hash';
+      const token = 'some-token';
+      const notification = 'new_review';
+      const params = {
+        api: state.api,
+        hash,
+        notification,
+        token,
+      };
+
+      mockApi
+        .expects('callApi')
+        .withArgs({
+          apiState: params.api,
+          auth: false,
+          endpoint: `accounts/unsubscribe`,
+          method: 'POST',
+          body: {
+            hash,
+            notification,
+            token,
+          },
+        })
+        .returns(createApiResponse());
```

Comment:
This API does return a specific response. Should we be specifying that, similar to what we do for the test for `updateUserNotifications`? I'm not sure what the value is of doing it in that test, but not in this test.

Reply:
I am not sure because the API function does not change the HTTP response and there is no assertion we can use here to guarantee that the code won't be obsolete if the real response changes.

I'd rather see proper responses in the saga layer because that's where the response data would be consumed.